src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppExistingBeanstalkEnvironment.recipe

```diff
@@ -30,15 +30,27 @@
             ],
             "Effect": {
                 "Pass": { "Include": true },
-                "Fail": {"Include": false}
+                "Fail": { "Include": false }
             }
         }
     ],
-
+    "Categories": [
+        {
+            "Id": "Hosting",
+            "DisplayName": "Hosting",
+            "Order": 20
```

Comment:
why do the categories order start with 20 unlike other recipes?

Reply:
Not a particular great reason other then this is the one recipe that didn't have a General category which in all of the others had the order set to 10.